package.json

```diff
@@ -25,12 +25,16 @@
     "body-parser": "^1.13.3",
     "cookie-parser": "^1.3.5",
     "debug": "^2.2.0",
+    "eslint": "^1.10.1",
     "express": "^4.13.2",
     "hogan-express": "^0.5.2",
-    "jshint": "^2.8.0",
+    "kerberos": "~0.0",
```

Comment:
What exactly is kerberos? Looked online and couldn't find much other than "C++ extension."